app/templates/suppliers/upload_countersigned_agreement.html

```diff
@@ -22,7 +22,7 @@
 
 {% block main_content %}
   {% if remove_countersigned_agreement_confirm %}
-      <form method="post" action='{{ url_for(".remove_countersigned_agreement_file", supplier_id=supplier.id, framework_slug=framework.slug) }}'>
+      <form method="post" action="{{ url_for(".remove_countersigned_agreement_file", supplier_id=supplier.id, framework_slug=framework.slug) }}">
```

Comment:
If we are going to do this everywhere in future, can we swap the inner quotes as well please? As in, `url_for ('.remove_countersigned_agreement_file', ...`